packages/teacher-app/public/locales/hi/translation.json

```diff
@@ -275,5 +275,11 @@
   "ENTER_WORKSHEET_DETAILS": "वर्कशीट विवरण दर्ज करें",
   "SHOW_QUESTIONS_BASED_ON": "पर आधारित प्रश्न दिखाएं",
   "PLEASE_SELECT_LIMIT": "कृपया सीमा चुनें",
-  "QUESTION_NOT_FOUND": "इस फ़िल्टर के लिए कोई प्रश्न नहीं मिला"
+  "QUESTION_NOT_FOUND": "इस फ़िल्टर के लिए कोई प्रश्न नहीं मिला",
+  "NO_STUDENT_HAS_ACHIEVED_ATTENDANCE_THIS_WEEK": "इस सप्ताह किसी भी छात्र ने 100% उपस्थिति हासिल नहीं की है",
```

Comment:
number can also be translated using utf codes

Reply:
how to do that ??